uber/templates/regform.html

```diff
@@ -32,9 +32,13 @@
         noCellphoneClicked();
         $.field('no_cellphone').on('click', noCellphoneClicked);
     });
-</script>
 
-<div id="form-begin-spacer" class="form-group"></div>
```

Comment:
This is where we remove the badge levels which are lower than the amount which already exists.